src/transformers/models/data2vec/data2vec_text.py

```diff
@@ -0,0 +1,474 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# TODO: Remove file
```

Comment:
This was copied from fairseq's codebase to help in the conversion of model weights and should be removed before merging this PR